distsql/select_result.go

```diff
@@ -163,6 +187,20 @@ func (r *selectResult) Next(ctx context.Context, chk *chunk.Chunk) error {
 	return nil
 }
 
+func (r *selectResult) readFromArrow(ctx context.Context, chk *chunk.Chunk) error {
+	if r.selectResp == nil || len(r.selectResp.RowBatchData) == 0 {
+		err := r.getSelectResp()
+		if err != nil || r.selectResp == nil {
+			return errors.Trace(err)
+		}
+	}
+	rowBatchData := r.selectResp.RowBatchData
+	codec := chunk.NewCodec(r.fieldTypes)
+	remained := codec.DecodeToChunk(rowBatchData, chk)
```

Comment:
we should put multiple `rowBatchData` into one `chk` in order to make it full.

Reply:
There are many details to judge `!chk.IsFull()` in arrow format.  Need a struct to maintain the `remained` . 
 I will do this in the next pr.

Reply:
okay